src/test/java/guitests/guihandles/ToDoListEventCardHandle.java

```diff
@@ -35,7 +35,14 @@ public String getTitle() {
     }
 
     public String getPriority() {
-        return priorityLabel.getText();
+        String priority =  priorityLabel.getText();
+        if (priority.contains("High")) {
+            return "H";
+        } else if (priority.contains("Medium")) {
+            return "M";
+        } else {
+            return "L";
+        }
```

Comment:
For this bug, I think we should probably refactor the Priority class instead.

I feel that it is more "correct" for the 'value' in the Priority class to be stored as "High", "Medium" or "Low" depending on the user input. This would probably cause a few other tests to fail also, but we need to properly refactor the code by week 13, so there's no other choice.

Ideally, we should use an enum to represent priority, but that would take even more work to modify everything.

Reply:
This one can change if have time when everything is done

Reply:
i don't think enum (vig's suggestion) is necessary since we only have 3 possibilities.